_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Custom Clock Processes
+nav: Custom Clock Processes
```

Comment:
nitpick: if I remember well, if the `nav` is the same as the `title` it is not useful to add it again
```suggestion
title: Custom Clock Processes
```